### DISCUSSION TITLE
Should this live under nvimtools?

### DISCUSSION THREAD

**Discussion by benlubas** [nvim-island/hydra.nvim#10] [General]
This fork is currently under an organization that exists purely for this plugin, seems like it was just made ad-hoc, so I'd like to bring up/discuss the option of letting this live under [nvimtools](https://github.com/nvimtools). It's the community org that handles none-ls. I'm not affiliated with them at all, and I've not discussed it with them. But it seems like this would fit under their umbrella nicely, and they already have a lot of eyeballs b/c null-ls was huge.

@miversen33

**Comment by miversen33** [comment]
I am certainly not against the idea, nvim-island was created specifically to hold abandoned plugins that are still used. Currently only hydra lives here and I would be more than happy to turn over maintenance of it to another team provided that team plans on keeping it alive. This seems like a discussion that should likely involve that team. I don't know any of the contributing members there and therefore I will not tag any of them here, however if you have connections, feel free to join them into this discussion. As it is, hydra is currently stable though could likely use some feature love. I am not super motivated to provide that love as really I just wanted it to not be dead.

**Reply by benlubas** [reply]
I don't think they would necessarily maintain it so much as you or I would maintain it (along with other community members that may want to join in) under the nvim tools umbrella.

I know the approach for none-ls is to just add a ton of maintainers b/c that project it so massive. I think for something like hydra, stability is a little more valuable, and having just one or two maintainers that are around for the occasional bug fix and to review PRs and stuff would be more than enough. If it gets out of hand we can always just hand it off to someone else.

@mochaap (nvimtools owner), I'd like to propose adding another beloved neovim plugin to your organization.

**Reply by mochaaP** [reply]
Is a forked repo and admin access on it enough for you? Let me know if you need more help. 😄

**Reply by benlubas** [reply]
Yeah that sounds like it would be enough. At the very least it's a good start and if it's not enough I can ping you again.

**Reply by benlubas** [reply]
@miversen33 the new repo exists. Both you and I have admin. I setup some branch protection rules, renamed master to main, opened the wiki to everyone (I think the wiki is the place to put most of the examples, not the README), and enabled issues.

thanks @mochaaP for the swift responses!

**Reply by miversen33** [reply]
I really appreciate your help, both you and @mochaaP  :)

I will get something pushed into this repo later today for a 2 month warning about deprecation and removal of this repo. I suppose I should also drop something in /r/neovim informing the rest of the world of the new permanent home of hydra. 

As I said in my initial forking, this org was never meant to hold it forever, just give it a temp home while a permanent home is figured out for it :)
